### DISCUSSION TITLE
sahi+ solo (mmdetection) not working

### DISCUSSION THREAD

**Discussion by zanetsc** [obss/sahi#617] [Q&A]
SAHI and solo_v2 from mmdetection is not working, objects are not detecting:
WARNING - sahi.model -   ignoring invalid prediction with bbox: [0. 0. 0. 0.]

```
detection_model = MmdetDetectionModel(
    model_path=model_path,
    config_path=config_path,
    confidence_threshold=0.2,
    device="cuda:0", # or 'cpu'
)
```

**Comment by fcakyon** [comment]
@zanetsc can you specify your mmdet, mmcv, cuda and sahi versions?

**Comment by zanetsc** [comment]
@fcakyon   I'm using:
mmdet = 2.25.1
mmcv = 1.6.1
torch &cuda 1.9.0+cu111 True
sahi =0.10.5

**Comment by fcakyon** [comment]
@zanetsc are you having any errors? Why do you think it is not working? Have you checked the visualizations?

**Comment by zanetsc** [comment]
When I'm using solo_v2 + sahi I do not receive correct results, instead of predictions I got: WARNING - sahi.model - ignoring invalid prediction with bbox: [0. 0. 0. 0.] (I havd  many of that warnings and lack of results). When I'm using solo_v2 from mmdetection (without sahi) I have correct results. When Im using other models than solo_v2 (such as maskrcnn) and sahi, than it is working well - I have correct detections. So, only for solo_v2 + sahi I do not have results but only warnings.

**Comment by fcakyon** [comment]
Thanks a lot for the detailed report! We will look into solov2 outputs from mmdetection.
